data/data/aws/vpc/sg-master.tf

```diff
@@ -148,6 +148,46 @@ resource "aws_security_group_rule" "master_ingress_kubelet_insecure" {
   self      = true
 }
 
+resource "aws_security_group_rule" "master_ingress_kube_scheduler_from_worker" {
+  type                     = "ingress"
+  security_group_id        = "${aws_security_group.master.id}"
+  source_security_group_id = "${aws_security_group.worker.id}"
+
+  protocol  = "tcp"
+  from_port = 10251
+  to_port   = 10251
+}
+
+resource "aws_security_group_rule" "master_ingress_kube_scheduler" {
```

Comment:
nit: the rest of this file leads with the implicit-from-master `resource` and then follows with the `_from_worker` resource.  Can you move this before the `master_ingress_kube_scheduler_from_worker` `resource`?

Reply:
on it